ui/raidboss/raidboss_alerts_only.html

```diff
@@ -1 +1,10 @@
-<meta http-equiv="refresh" content="0; url=raidboss.html?timeline=0&alerts=1" />
\ No newline at end of file
+<script type="text/javascript">
+  let redirUrl = 'raidboss.html?timeline=0&alerts=1';
+  if (typeof window !== 'undefined') {
+    const wsUrl = /[\?&](OVERLAY_WS=[^&]+)/.exec(window.location.href);
```

Comment:
Would it make sense to preserve all the params minus timeline and alerts via `URLSearchParams` or something?

Reply:
Possibly, though I don't know of any occasion where any other params would happen. This is only needed because the WS URL generator appends OVERLAY_WS and that causes problems for people that want to use it for streaming or XL BrowserHost.

If people are adding their own params to the URL, they're probably capable of using the raidboss.html page with params instead?

Reply:
Sure, agreed that it's unlikely.  It just seemed better to not have to parse urls manually and to handle future edge cases at the same too.  I think this is an improvement for now, so will merge.